app/assets/javascripts/discourse/app/services/breadcrumbs-service.js

```diff
@@ -0,0 +1,40 @@
+import { tracked } from "@glimmer/tracking";
+import { warn } from "@ember/debug";
+import Service from "@ember/service";
+
+export default class BreadcrumbsService extends Service {
+  @tracked containers = [];
+  #containers = [];
```

Comment:
This is taken from the original library -- I am pretty sure this is to prevent rerender errors in Ember where, if you modify a tracked array that some other component relies on, you can end up in an infinite render loop, and Ember throws an error to stop this. Will go double check this locally.